src/TwigComponent/src/CVA.php

```diff
@@ -12,107 +12,124 @@
 namespace Symfony\UX\TwigComponent;
 
 /**
- * @author Mathéo Daninos <matheo.daninos@gmail.com>
+ * Class Variant Authority (CVA) resolver.
+ *
+ * The CVA concept is used to render multiple variations of components, applying
+ * a set of conditions and recipes to dynamically compose CSS class strings.
+ *
+ * @see https://cva.style/docs
  *
- * CVA (class variant authority), is a concept from the js world.
- * https://cva.style/docs
- * The UI library shadcn is build on top of this principle
- * https://ui.shadcn.com
- * The concept behind CVA is to let you build component with a lot of different variations called recipes.
+ * @doc https://symfony.com/bundles/ux-twig-component/current/index.html
+ *
+ * @author Mathéo Daninos <matheo.daninos@gmail.com>
  *
  * @experimental
  */
 final class CVA
 {
     /**
-     * @var string|list<string|null>|null
-     * @var array<string, array<string, string|list<string|null>>|null the array should have the following format [variantCategory => [variantName => classes]]
-     *                                                ex: ['colors' => ['primary' => 'bleu-8000', 'danger' => 'red-800 text-bold'], 'size' => [...]]
-     * @var array<array<string, string|array<string>>> the array should have the following format ['variantsCategory' => ['variantName', 'variantName'], 'class' => 'text-red-500']
-     * @var array<string, string>|null
+     * @var list<string>
```

Comment:
Are you sure list<string> include null values? Maybe list<string|null>

Reply:
What is the use case to pass null values ?

Reply:
https://github.com/symfony/ux/issues/1570

Reply:
Could we filter those values in the Twig function and/or the constructor ?

Reply:
Actually we test for null so I guess your code already works with nullable 😉

Reply:
Ok so just update the docblock, right ?

Reply:
Updated!